app/code/Magento/Customer/composer.json

```diff
@@ -25,7 +25,8 @@
         "magento/module-tax": "*",
         "magento/module-theme": "*",
         "magento/module-ui": "*",
-        "magento/module-wishlist": "*"
+        "magento/module-wishlist": "*",
+        "magento/module-persistent": "100.2.*"
```

Comment:
This does not make any sense as module `Persistent` depends on `Customer`: https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Persistent/composer.json#L12

Reply:
Hello @orlangur ,

Thank you for reviewing this fix.

I would like to let you know that, If we disable this extension than whenever customer add product into the cart, than they will always get new quote id, that's why persistence extension needed dependency with the customer.

Please have a look at this observer class: Magento\Persistent\Observer\SetQuotePersistentDataObserver

In this observer, Whenever customer add product into the cart they will have a quote id that is generated first time, than if customer again add any product into the cart than that product will add to cart with same quote id because of this persistence extension's observer. So persistence connection needed.

Now, if we disable that so customer always get a new quote id whenever they add product into the cart.

Hope you got my point, Please do let me know if you have any question regarding that.

Thanks,

Hiren

Reply:
@hiren0241 `Customer` module cannot depend on `Persistent` module. Please elaborate another fix.